### PR TITLE
Remove pre-spa days code from flow list view

### PR DESCRIPTION
**Pull request by rowanseymour**
Also removing the empty_include blurb because pretty much nobody ever sees it on account of all workspaces getting sample flows.

### CODE REVIEW

File: temba/flows/views.py [hunk 1]
```diff
@@ -669,19 +669,13 @@ def update_triggers(self, flow, user, new_keywords: list):
...
         default_template = "flows/flow_list.html"
         default_order = ("-saved_on",)
         search_fields = ("name__icontains",)
 
         def get_context_data(self, **kwargs):
             context = super().get_context_data(**kwargs)
-            context["org_has_flows"] = self.request.org.flows.filter(is_active=True).exists()
-            context["folders"] = self.get_folders()
-            context["labels"] = self.get_flow_labels()
-            context["campaigns"] = self.get_campaigns()
```
Comment (rowanseymour) [inline temba/flows/views.py:683]: these used to be part of the side menu.. but that's a separate view now

File: temba/flows/views.py [hunk 2]
```diff
@@ -669,19 +669,14 @@ def update_triggers(self, flow, user, new_keywords: list):
...
-        refresh = 10000
         fields = ("name", "modified_on")
         default_template = "flows/flow_list.html"
         default_order = ("-saved_on",)
         search_fields = ("name__icontains",)
 
         def get_context_data(self, **kwargs):
             context = super().get_context_data(**kwargs)
-            context["org_has_flows"] = self.request.org.flows.filter(is_active=True).exists()
-            context["folders"] = self.get_folders()
```
Comment (rowanseymour) [inline temba/flows/views.py:681]: these used to be part of the side menu.. but that's a separate menu view now

File: temba/flows/views.py [hunk 3]
```diff
@@ -669,19 +669,14 @@ def update_triggers(self, flow, user, new_keywords: list):
...
         title = _("Flows")
-        refresh = 10000
         fields = ("name", "modified_on")
         default_template = "flows/flow_list.html"
         default_order = ("-saved_on",)
         search_fields = ("name__icontains",)
 
         def get_context_data(self, **kwargs):
             context = super().get_context_data(**kwargs)
-            context["org_has_flows"] = self.request.org.flows.filter(is_active=True).exists()
```
Comment (rowanseymour) [inline temba/flows/views.py:680]: orgs pretty much always have flows..

File: temba/flows/views.py [hunk 4]
```diff
@@ -669,19 +669,14 @@ def update_triggers(self, flow, user, new_keywords: list):
...
-            context["folders"] = self.get_folders()
-            context["labels"] = self.get_flow_labels()
-            context["campaigns"] = self.get_campaigns()
-            context["request_url"] = self.request.path
+            context["labels"] = self.request.org.flow_labels.order_by(Lower("name"))
```
Comment (rowanseymour) [inline temba/flows/views.py:679]: This seems to still be needed for bulk labeling to work.. probably should be using `get_bulk_action_labels` but I'l look into that separately

File: templates/flows/flow_filter.html
Comment (rowanseymour) [inline templates/flows/flow_filter.html:1]: not used

File: temba/flows/views.py [hunk 5]
```diff
@@ -669,19 +669,14 @@ def update_triggers(self, flow, user, new_keywords: list):
     class BaseList(BulkActionMixin, ContextMenuMixin, BaseListView):
         permission = "flows.flow_list"
         title = _("Flows")
-        refresh = 10000
```
Comment (rowanseymour) [inline temba/flows/views.py:672]: no longer used

**Review by norkans7 — APPROVED** [review]

**Pull request merged**